example/storybook/src/example-screens/components/SkeletonComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Let's create two different components. SkeletonCircle & SkeletonBox instead of changing styles based on props.